### PR TITLE
Make logging to console option

### PR DESCRIPTION
**Pull request by JoeZiminski**
In some instances you may want to log to file but not to console, so added an option for this.

### CODE REVIEW

File: fancylog/fancylog.py [hunk 1]
```diff
@@ -60,7 +61,8 @@ def start_logging(
     :param log_to_file: If True, write a log file, otherwise just print to
     terminal.
     :param timestamp: If True, add a timestamp to the filename
-    :return: Path to the logging file
+    :param log_to_console: Print logs in stderr or not: Default: True
```
Comment (adamltyson) [inline fancylog/fancylog.py:64]: Would this not be to stdout?

**Review by adamltyson — APPROVED** [review]

File: fancylog/fancylog.py (hunk 1 above)
Comment (JoeZiminski) [reply fancylog/fancylog.py:64]: that's correct, I renamed the function as realised this but forgot to update the doc, thanks!

**Comment by adamltyson** [comment]
@JoeZiminski looks good, but could you sort out the failing tests?

**Comment by JoeZiminski** [comment]
Thanks this should fix it

**Comment by adamltyson** [comment]
Thanks @JoeZiminski. This will be in `0.3.0` which will be released later today.

**Pull request merged**